bootstrap/etc/salt/master

```diff
@@ -24,6 +24,7 @@ gitfs_remotes:
   - https://github.com/mitodl/consul-formula
   - https://github.com/mitodl/vault-formula
   - https://github.com/mitodl/fluentd-formula
+  - https://github.com/mitodl/netdata-formula
```

Comment:
This isn't needed in the bootstrap. That's just for deploying new master servers, so it should be as barebones as we can make it.